unittests/resources/sxfm_example.sxfm

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE note [
+<!ELEMENT feature_model (meta?, feature_tree, constraints?)>
+<!ATTLIST feature_model
+    name CDATA #IMPLIED
+>
+<!ELEMENT meta (data*)>
+<!ELEMENT data (#PCDATA)>
+<!ATTLIST data name CDATA #REQUIRED>
+<!ELEMENT feature_tree (#PCDATA)>
+<!ATTLIST feature_tree
+    indentation_string CDATA #IMPLIED
+>
+<!ELEMENT constraints (#PCDATA)>
+]>
+<feature_model name="My feature model">
+  <meta>
+  <data name="description">Model description</data>
+  <data name="creator">Model's creator</data>
+  <data name="email">Model creator's email</data>
+  <data name="date">Model creation date</data>
+  <data name="department">Model creator's department</data>
+  <data name="organization">Model creator's organization</data>
+  <data name="address">Model creator's address</data>
+  <data name="phone">Model creator's phone</data>
+  <data name="website">Model creator's website</data>
+  <data name="reference">Model's related publication</data>
+  </meta>
+  <feature_tree>
+    :r root (root_id)
+      :o opt1 (id_opt1)
+      :o opt2 (id_opt2)
+      :m man1
```

Comment:
is it allowed that mandatory features have no (display-)name ?

Reply:
If you are referring to the missing ID: yes, this is allowed. 
To be honest, I would simply enforce that the names have to be unique and give each of the features the id 'id_<name>'. This would simplify a lot...
@Kallistos What do you think?

Reply:
That depends what the names of the features are.
If the feature name is the unique part right before the parenthesis, that is okay for me.
In SPLConqueror this is different right now. There the part in parenthesis is the name that is used as feature name.

Reply:
Correct, the feature name is the unique part right before the parenthesis. 
Yes, you are right regarding SPL Conqueror. However, I do not intend to implement is exactly as it is implemented in SPL Conqueror, as the implementation is partially error-prone.

Reply:
Btw. the mandatory feature above is from the original SXFM example of the SPLOT repository (http://www.splot-research.org/).

Reply:
Okay!
Then the Id part in the parenthesis seems pretty useless anyways, right?

Reply:
Under the assumption that the feature names are already unique: yes, definitely!